### ISSUE TITLE
Support gradle build cache

### ISSUE THREAD

**Issue by stephanenicolas** [evant/gradle-retrolambda#258]
It looks like the retro lambda tasks are not cacheable by gradle:
https://docs.gradle.org/3.5/userguide/build_cache.html

**Comment by evant** [comment]
Hm, shouldn't this be the job of the transform api to implement?

**Comment by stephanenicolas** [comment]
Sorry, I actually realized that the new android plugin version 3.0 allows
to remove retro lambda. So, we don't need this feature anymore from retro
lambda.

But in case you want to implement it, it's on your side if you create a new
type of task.

2017-08-09 13:31 GMT-07:00 Evan Tatarka <notifications@github.com>:

**Comment by evant** [comment]
Yeah, 3.0 is going to be the last version I plan on supporting. Unfortunately by using the transform api I don't have control of the task itself. I might look into it for the plain java version though.

**Comment by stephanenicolas** [comment]
Even though we are probably gonna stop using retro lambda, I would like to
take some time to thank you for bringing this to android.
Thanks a lot for all your work !

2017-08-09 14:15 GMT-07:00 Evan Tatarka <notifications@github.com>:
